fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraVersioningIT.java

```diff
@@ -1714,7 +1633,6 @@ private void createVersionedExternalBinaryMemento(final String rescId, final Str
         final HttpPut httpPut = putObjMethod(rescId);
         httpPut.addHeader(LINK, "<" + NON_RDF_SOURCE.getURI() + ">;rel=\"type\"");
         httpPut.addHeader(LINK, getExternalContentLinkHeader(externalUri, handling, null));
-        httpPut.addHeader(LINK, VERSIONED_RESOURCE_LINK_HEADER);
```

Comment:
The declaration of the constant at the top of this class can now be removed. (`VERSIONED_RESOURCE_LINK_HEADER`)